__test__/integration/debt_token_api/runners/transfer_base.ts

```diff
@@ -68,33 +66,38 @@ export abstract class TransferBaseScenarioRunner extends ScenarioRunner {
                     MALFORMED_TOKEN_RECIPIENT,
                 );
 
-                const tokenIDs = await Promise.all(
-                    scenario.orders.map(this.generateDebtTokenForOrder),
+                const orderOneTokenID = await this.generateDebtTokenForOrder(
+                    scenario.orderFilledByCreditorOne,
                 );
 
-                const [creditorsTokenID, nonCreditorsTokenID] = tokenIDs;
+                const orderTwoTokenID = await this.generateDebtTokenForOrder(
+                    scenario.orderFilledByCreditorTwo,
+                );
 
                 // Transfer nonCreditorsToken to a different address
                 await debtTokenAPI.transferFrom(
-                    scenario.creditor,
-                    NON_CREDITOR_ADDRESS,
-                    nonCreditorsTokenID,
+                    scenario.orderFilledByCreditorTwo.creditor,
```

Comment:
This whole `transferFrom` block is no longer necessary now that the `nonCreditorsTokenID` is filled by `creditorTwo` -- please remove.